board/safety/safety_ford.h

```diff
@@ -9,22 +9,25 @@
 
 int ford_brake_prev = 0;
 int ford_gas_prev = 0;
-int ford_is_moving = 0;
+bool ford_moving = false;
 
 static void ford_rx_hook(CAN_FIFOMailBox_TypeDef *to_push) {
 
   int addr = GET_ADDR(to_push);
 
   if (addr == 0x217) {
     // wheel speeds are 14 bits every 16
-    ford_is_moving = 0xFCFF & (to_push->RDLR | (to_push->RDLR >> 16) |
-                               to_push->RDHR | (to_push->RDHR >> 16));
+    ford_moving = false;
+    for (int i = 0; i < 8; i += 2) {
+      int j = i + 1;  // Misra 10.8: can't cast (i+1) directly in uint
```

Comment:
Don't do this!

Reply:
solution is still a bit dumb, but won't trigger Misra